uyuni/uyuni-storage-setup/scripts/mgr-storage-server

```diff
@@ -40,7 +40,7 @@ if [ ! $UID = 0 ];then
 fi
 
 if [ ! -x /usr/bin/mgradm ]; then
-    die "$0 support only SUSE Manager 5.0 or later"
+    die "$0 support only SUSE Multi-Linux Manager 5.0 or later"
```

Comment:
should this be 5.1 or 5.0? I mean, 5.0 is  still called SUSE Manager  :thinking: 
```suggestion
    die "$0 support only SUSE Multi-Linux Manager 5.1 or later"
```

Reply:
I was mainly taking care of the name but yes, I can change this too.